lib/fungus_toast/games/games.ex

```diff
@@ -94,6 +95,13 @@ defmodule FungusToast.Games do
 
       game
       |> Players.create_ai_players(ai_player_count)
+      
+      # if(human_player_count == 1) do
```

Comment:
I think the work of creating a grid should maybe be in a `start_game` function, and that can be called here or from another REST request

Reply:
Makes sense. Would start_game get called every time and be responsible for checking whether the game really can start (i.e. all human players have users assigned)?

Reply:
That makes sense to me